app/views/activities/_activity.html.slim

```diff
@@ -16,8 +16,14 @@
     .pull-left.name
       = activity.name
     .pull-right.icon
-      - if current_user.completed_activity?(activity)
-          i.fa.fa-check-square-o
-      - else
-          i.fa.fa-square-o
+      -if current_user.is_a? Student
+        - if current_user.completed_activity?(activity)
+            i.fa.fa-check-square-o
+        - else
+            i.fa.fa-square-o
+      -else
+        div.completed_activities  
+          = cohort.students.select { |student| student.completed_activity?(activity) }.count
```

Comment:
Querying in the view isn't a great idea. Maybe make a helper method that takes a cohort and an activity and returns the completed/total numbers. Try to have the database do the aggregation instead of doing in memory.